NullPointerException in deserialization

Version:  3.4-beta-1-forge-1.12.2

Server crashes when DynmapBlockScan runs (modpack: Divine Journey 2).

Stack trace of crash report:
```
net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from DynmapBlockScan (dynmapblockscan)
Caused by: java.lang.NullPointerException
    at org.dynmapblockscan.core.blockstate.Variant$Deserializer.deserialize(Variant.java:131)
    at org.dynmapblockscan.core.blockstate.Variant$Deserializer.deserialize(Variant.java:112)
    ....
```

Cause seems to be a typing error in:
https://github.com/webbukkit/DynmapBlockScan/blob/583ebd9b959c841f44d0a9bf04b02d03f336489b/core/src/main/java/org/dynmapblockscan/core/blockstate/Variant.java#L131

-> 'y' should be 'z'

--- a/core/src/main/java/org/dynmapblockscan/core/blockstate/Variant.java
+++ b/core/src/main/java/org/dynmapblockscan/core/blockstate/Variant.java
@@ -128,7 +128,7 @@ public class Variant {
                 y = obj.get("y").getAsInt();
             }
             if (obj.has("z")) {
-                z = obj.get("y").getAsInt();
+                z = obj.get("z").getAsInt();
             }
             var.rotation = ModelRotation.getModelRotation(x, y, z);
     		if (obj.has("model"))

